.github/workflows/publish-to-pypi.yml

```
@@ -0,0 +1,27 @@
+name: Publish releases to PyPI
+
+on:
+  release:
+    types: [published, prereleased]
+
+jobs:
+  build-and-publish:
+    name: Builds and publishes releases to PyPI
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@main
```

```suggestion
      - uses: actions/checkout@v2.4.0
```

https://github.com/actions/checkout/releases/tag/v2.4.0